pkg/service.pb/metadata.pb.go

```diff
@@ -0,0 +1,844 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: metadata.proto
+
```

Comment:
Here two should be ignore too.

Reply:
the metadata.pb.go will be used in unit test, for check config and db orm.

Reply:
the generated metadata.validator.pb.go is empty, it is harmless.
if we need ignore it, we must change the generate code command.